android/assets/html-zh-rCN/about1d.html

```diff
@@ -9,7 +9,7 @@
 <p> 传统的条形码，也被称为一维码。常见的条形码有几种（比如 UPC 和 EAN）。下面是一种常见的条形码： </p>
 <p class="imgcenter"><img src="../images/big-1d.png"/></p>
 <p> 条形码内含有一些数字或字符。商品或书本背后通常就会有条形码，并且每种商品的条码都不相同。你可以搜索商品条码来获得相应商品的价格、评论等信息。</p>
-<p> 如果扫描了图书的条码，还可以使用 Google Book Search 在这本书里找某个单词或短语在哪一页。在中国大陆由于特殊国情暂时无法使用此功能。 </p>
+<p> 如果扫描了图书的条码，还可以使用谷歌图书搜索在这本书里找某个单词或短语在哪一页。在中国大陆，由于特殊国情暂时无法使用此功能。 </p>
```

Comment:
I'd rather leave this as a direct translation of the English text, rather than add notes about availability in particular locations. It can change, and isn't strictly speaking locale-specific. (Here it happens to be almost exactly lined up with a locale though.)